CMakeLists.txt

```
@@ -82,16 +82,33 @@ macro(build_mimick)
 
 endmacro()
 
-build_mimick()
+option(SKIP_MIMICK
+  "When ON, mimick is not going to be build and mimick based tests are not going to be run"
```

@ivanpauno nit
```suggestion
  "When ON, mimick is not going to be built"
```
The second half of the sentence jumps ahead of itself a bit.